src/load_manager.cc

```diff
@@ -272,6 +270,20 @@ LoadManager::StopWorkerThreads()
   threads_.clear();
 }
 
+void
+LoadManager::WaitForWarmupAndCleanup()
+{
+  for (std::thread& thread : threads_) {
+    if (thread.joinable()) {
+      thread.join();
+    }
+  }
+  threads_.clear();
+  threads_config_.clear();
+  threads_stat_.clear();
+  workers_.clear();
```

Comment:
These are the four data structures that all load managers share that need to be reset before actual benchmarking begins.